addons/mass_mailing_account/models/account_move.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+    _mailing_enabled = True
+
+    def _mailing_get_default_domain(self, mailing):
```

Comment:
It could probably be added in base account module (there is a '# MAIL.THREAD' section in code) (as well as the '_mailing_enabled' class attribute). It would not be used until mass_mailing is installed, but it allows to avoid having a bridge module just for a class attribute and a method.